### ISSUE TITLE
1.21.1+ support?

### ISSUE THREAD

**Issue by kristibektashi** [Ladysnake/Requiem#633]
When will support for 1.20.1 be added?

Edit: Changed to 1.21.1 since 1.20.1 support has already been added (even though it's broken), and also because 1.20.1 is kinda old, see [below comment](https://github.com/Ladysnake/Requiem/issues/633#issuecomment-3225381338) for more info

**Comment by Redfan2** [comment]
This is a question for @Pyrofab

**Comment by Pyrofab** [comment]
The port has not been started yet, so not before next week at least

**Comment by kristibektashi** [comment]
Any progress?

**Comment by kristibektashi** [comment]
Still no port...

**Comment by Redfan2** [comment]
> Still no port...

Please note that a mod the size of Requiem takes time to port. Pyrofab has been working behind the scenes updating all the libraries Requiem relies on (Automatone, Blabber, Cardinal Components API, Fiber, Impersonate, Locki, PlayerAbilityLib and Satin).

**Comment by adaliszk** [comment]
If you feel that your issue needs to progress faster, often the best way to help that is to clone the repository and try to implement things yourself! While I have yet to merge any similar endeavors in the modding community, the task itself seems straightforward: bumping package versions and going through Minecraft code changes one by one. Even if the maintainers eventually catch up with you (and do a more advanced version of the job), you can learn some skills with the exercise!

However, given that the latest tagged versions are flagged as Beta, it might be challenging to upgrade everything since there are more significant changes in progress! I understand the frustration. I am waiting on the port myself, but I am trying to collaborate with the mods that are blockers. If Requiem is that blocker for you, it's your turn to investigate and collaborate!

**Comment by kristibektashi** [comment]
> > Still no port...
> 
> Please note that a mod the size of Requiem takes time to port. Pyrofab has been working behind the scenes updating all the libraries Requiem relies on (Automatone, Blabber, Cardinal Components API, Fiber, Impersonate, Locki, PlayerAbilityLib and Satin).

You are right and I completely understand what you're saying. I apologise for seeming to be impatient, I was really tired when I wrote that comment. I fully appreciate the effort that Pyrofab has put into this mod, and I am hoping that eventually it will support the latest version, no matter how long it takes.

**Comment by cph101** [comment]
I will try my best to port as much as I can, but no guarantees. I do think it is priority to fix survival-breaking glitches such as #634, #636 and #637 first.

**Comment by danandsem** [comment]
can we get requiem 1.20.1 port before GTA 6?
PS: joke

**Comment by ServiceGames** [comment]
how unfortunate

**Comment by benix268** [comment]
hi! how's requiem coming along?

**Comment by kristibektashi** [comment]
> hi! how's requiem coming along?

~~It's my understanding that it's still as dead as it has been for the last 3 years, which is unfortunate since it's one of the best mods I have ever played. However it's up to the devs if they want to port it, and I really hope they do but wouldn't get my hopes up.~~

~~Like the comments above me are saying, perhaps it's about time someone else creates a fork and ports it over themselves (I don't have the skillset required to do it myself unfortunately).~~

~~Also, since 1.21.1 is now the de-facto "LTS" version of Minecraft, aka the version that most modders stick to because the game updates too fast to keep up with the new game drops , I'm changing the title of this issue to that version.~~

Edit: Actually cancel that, 1.20.1 support HAS actually already happened in the `1.20` branch (https://github.com/Ladysnake/Requiem/tree/1.20), and I'm surprised this issue wasn't closed along with the others (https://github.com/Ladysnake/Requiem/issues/654, https://github.com/Ladysnake/Requiem/issues/648) considering it's the oldest one. However, it apparently does not build properly, and is therefore broken, according to https://github.com/Ladysnake/Requiem/issues/676.

I'm still repurposing this issue for a potential 1.21.1+ port by changing the title and editing the original issue body. For 1.20.1 improvements the correct place to ask would be https://github.com/Ladysnake/Requiem/issues/676.
